packages/get-starknet/.eslintrc.js

```diff
@@ -38,5 +38,5 @@ module.exports = {
     },
   ],
 
-  ignorePatterns: ['!.eslintrc.js', 'dist/', '**/test', '.nyc_output/', 'coverage/'],
+  ignorePatterns: ['!.eslintrc.js', 'dist/', '**/test', '.nyc_output/', 'coverage/', '*.js'],
```

Comment:
may be just for webpack?

as in root directory, we already has a rule to check js file

```suggestion
  ignorePatterns: ['!.eslintrc.js', 'dist/', '**/test', '.nyc_output/', 'coverage/', 'webpack.*.js'],
```